hypha/static_src/src/app/src/containers/ReminderContainer/index.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import injectReducer from '@utils/injectReducer'
+import injectSaga from '@utils/injectSaga'
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators, compose } from 'redux';
+import PropTypes from 'prop-types';
+import * as Actions from './actions';
+import reducer from './reducer';
+import saga from './sagas';
+import * as Selectors from './selectors';
+import "./styles.scss";
+import { SidebarBlock } from '@components/SidebarBlock'
+import LoadingPanel from '@components/LoadingPanel';
+import Modal from '@material-ui/core/Modal';
+import { withStyles } from '@material-ui/core/styles';
+import ReminderList from './components/ReminderList';
+import ReminderForm from './containers/ReminderForm' 
+
+
+const styles = {
+    modal: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+    },
+  };
+
+class ReminderContainer extends React.PureComponent {
+
+    state = {
+        open : false
+    }
+
+    componentDidUpdate(prevProps) {
+        if(this.props.submissionID != prevProps.submissionID) {
+            this.props.initAction(this.props.submissionID)
+        }
+    }
+
+    handleModalClose = () => {
+        this.setState({open : false})
+    }
+
+    render(){
+        const { classes } = this.props;
+        return (
+            this.props.reminderInfo.loading 
+            ? <LoadingPanel /> :
+            <div className="reminder-container">
```

Comment:
What do you think about moving the <LoadingPanel /> outside the main `return` block.

We can put an early-return to avoid nested contions.